### DISCUSSION TITLE
Add timezone option

### DISCUSSION THREAD

**Discussion by eplusresearch** [longhorn/longhorn#8963] [Ideas]
My system is TimeZone 'Asia/Ho_Chi_Minh'. In the future, can LONGHORN support the following options?
 extraEnvVars:
 - name: TZ
   value: 'Asia/Ho_Chi_Minh'

**Comment by derekbit** [comment]
I've received [a similar request from another user](https://github.com/longhorn/longhorn/issues/8754).
@innobead WDYT?
